worlds/bluerov2_underwater.world

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <sdf version="1.6">
-  <world name="underwater">
+  <world name="bluerov2_underwater">
```

Comment:
Hmmm... we could drop the _underwater suffix to make the world names a bit shorter.

Reply:
Do you think that the filename should be updated as well? It looks like the [SITL_Models](https://github.com/ArduPilot/SITL_Models/tree/master/Gazebo/worlds) repo prefers to keep the environment descriptor in the filename.

Reply:
Hmm... let's keep the same name.